site/usage/webcomponent.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: usage
+menu: usage
+title: Embedding Vega-Lite with vega-element
+permalink: /usage/webcomponent.html
+---
+
+[Web components]((https://www.webcomponents.org/introduction) are a set of web platform APIs that allow you to create new custom, reusable, encapsulated HTML tags to use in web pages and web apps.
+
+[`vega-element`](https://www.webcomponents.org/element/PolymerVis/vega-element) is a Polymer webcomponent that can be used to embed vega or vega-lite visualization using custom HTML tags.
+
+**Quick start**
+The simplest usage case is to just embed a vega-lite visualization.
+```html
+<!-- import vega-element webcomponent from CDN (not for production use!!) -->
```

Comment:
Can you put vega-element on a proper CDN like cdnjs?

Reply:
Not enough stars for the lib to be included in cdnjs - they have a popularity requirement.

Do u have any recommendations for any cdn with lower requirements?

Reply:
Hmm, in that case I'd document how to add the required sources with npm.